packages/aws-cdk/README.md

```diff
@@ -560,6 +561,111 @@ This feature is currently in preview. Be aware of the following limitations:
   in the right order. The CLI will not help you import dependent resources in the right
   order, the CloudFormation deployment will fail with unresolved references.
 
+
+### `cdk migrate`
+
+⚠️**CAUTION**⚠️ 
+
+CDK migrate is currently experimental and may have breaking changes in the future. 
+
+Generates a CDK application from an existing JSON/YAML CloudFormation template.
+Takes a valid CloudFormation template as input from either a local file specified with `--from-path`,
+or from a deployed Cloudformation stack using `--from-stack`. Generates a CDK application 
+which will synthesize a CloudFormation template with identical resource configurations to the provided template. 
+The generated application will be initialized in the current working directory with a single stack where 
+the stack, app, and directory will all be named using the provided `--stack-name`. The generated application will 
+be within a generated subdirectory in your current working directory unless `--output-path` is specified. 
+All CDK supported lanaguages are supported, language choice can be specified with `--language`.
+
+#### Generate a typescript application from a local template.json file
```

Comment:
This is very good, nice work. I only have small comment; can we parse yaml templates as well, or is it json only? This doc should clarify.

Reply:
I added a bit above this about it supporting JSON/YAML